core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -659,6 +659,13 @@ public enum CoreError implements ScalarDbError {
       "The variable key column size must be greater than or equal to 64",
       "",
       ""),
+  COSMOS_PRIMARY_KEY_CONTAINS_ILLEGAL_CHARACTER(
+      Category.USER_ERROR,
+      "0145",
+      "The value of the column %s in the primary key contains an illegal character. "
+          + "Primary-key columns must not contain any of the following characters in Cosmos DB: ':', '/', '\\', '#', '?'. Value: %s",
```

Comment:
I might miss your point, but can we do something like below?
It might be too much, but we can avoid managing the invalid characters in two places.

```suggestion
      "The value of the column %s in the primary key contains an illegal character. "
          + "Primary-key columns must not contain any of the following characters in Cosmos DB: "
          + Chars.asList(CosmosOperationChecker.ILLEGAL_CHARACTERS_IN_PRIMARY_KEY).stream()
              .map(c -> "'" + c + "'")
              .collect(Collectors.joining(", "))
          + ". Value: %s";
```

Reply:
@feeblefakie As discussed, we will leave it as is. This is because we should avoid creating a dependency from the Cosmos adapter to CoreError, as each adapter will be separated into its own sub-module in the future.